.devcontainer/devcontainer.json

```diff
@@ -70,10 +70,10 @@
 		// custom
 		"ghcr.io/devcontainers/features/sshd:1": {},
 		"ghcr.io/devcontainers/features/docker-outside-of-docker:1": {},
-		"ghcr.io/devcontainers/features/kubectl-helm-minikube:1": {},
+		"ghcr.io/devcontainers/features/kubectl-helm-minikube:1.2.1": {}, // 1.2.2 seems to be broken
```

Comment:
nit: add TODO to the comment, to circle back on?

Reply:
We have dependabot on this, which should be bugging us